spec/config.json

```diff
@@ -421,6 +421,61 @@
         }
       }
     },
+    "client.tls": {
```

Comment:
Can you please move `tls` to a separate sub key? :) Dots in key names are not handled very well by the tooling.

Reply:
Thinking about this a bit more. I think the configuration key should be more explicit. There are two options:

- Support mTLS globally, so use it in all the clients (e.g. webhooks, backchannel, ...)
- Have mTLS configured per feature

In the first case, the current structure is probably ok. Maybe it makes sense to have a structure like:

```
http:
  client:
    default:
      tls:
      # in the future:
      # timeout ....
   backchannel_logout:
     tls:
       # ...
   refresh_token_webhook:
     # ...
```

WDYT?

Reply:
@aeneasr implemented per feature configuration with default fallback